entrypoint.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 MSRV_RESULT=$(cargo msrv --path $1 --output-format json | tail -1)
 MSRV=$(echo $MSRV_RESULT | jq -r '.msrv')
-echo "::set-output name=msrv::$MSRV"
+echo "{msrv}={$MSRV}" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
echo "msrv=$MSRV" >> $GITHUB_OUTPUT
```
Pretty sure the braces were just to show that it's a placeholder value :slightly_smiling_face: